client/i18n/locales/japanese/motivation.json

```diff
@@ -0,0 +1,819 @@
+{
```

Comment:
When we go to translate these, we can actually replace them with phrases and quotes that are more culturally relevant - we don't have to do exact translations (which is why this isn't on Crowdin).